solidity/src/Pragma.sol

```diff
@@ -47,7 +66,7 @@ contract Pragma is IPragma, PragmaDecoder {
 
     /// @inheritdoc IPragma
     function getUpdateFee(bytes[] calldata updateData) external view returns (uint256 feeAmount) {
-        // Get the update fee.
+        return 0;
```

Comment:
create issue so we don't forget to update that

Reply:
Done in #40